src/app/upgrade.h

```diff
@@ -91,6 +93,16 @@ bool upgradeResumeFile(const QString &filepath)
     fastNew = fastOld;
 
     int priority = fastOld.dict_find_int_value("qBt-queuePosition");
+    if (priority > maxPrio)
+        maxPrio = priority;
+
+    fastNew["qBt-savePath"] = Utils::String::toStdString(oldTorrent.value("save_path").toString());
+    fastNew["qBt-ratioLimit"] = Utils::String::toStdString(QString::number(oldTorrent.value("max_ratio", -2).toReal()));
+    fastNew["qBt-label"] = Utils::String::toStdString(oldTorrent.value("label").toString());
+    fastNew["qBt-name"] = Utils::String::toStdString(oldTorrent.value("name").toString());
+    fastNew["qBt-seedStatus"] = oldTorrent.value("seed").toBool();
+    fastNew["qBt-tempPathDisabled"] = false;
+
```

Comment:
The question for the all the above is: What is less likely to be corrupt? The .ini file or the .fastresume. Ideally they both have the same value.
I vote to trust the fastresumes.